qemu/tests/x86_cpu_flag_nonstop_tsc.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+
+from provider.cpu_utils import check_cpu_flags
+from provider.cpu_utils import cpu_check_cmd_output
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flag nonstop_tsc.
+    1) Check if current flags are in the supported lists, if no, cancel test
```

Comment:
It's checking if host supports this flag, right? Let's make the step clearer.

Reply:
update and thanks